src/mem_config.rs

```diff
@@ -34,7 +30,7 @@ pub trait MemConfig {
 }
 
 /// Linear growth, all segments have the same (FACTOR) length.
-pub struct Linear<const FACTOR: usize>;
+pub struct Linear<const FACTOR: usize = 1024>;
```

Comment:
My gut feeling is that this is large for a default value, was there any specific reason for this selection?